README.md

```diff
@@ -41,7 +41,8 @@ For running and testing locally one would need to create PostgreSQL databases
 
     createuser -R -S -D pubserver
     createdb -O pubserver pubserver
-    createdb -O pubserver pubserver_test
+    createdb -O pubserver pubserver_test 
+    echo 'CREATE EXTENSION IF NOT EXISTS "uuid-ossp";' | psql pubserver_test 
```

Comment:
This should be done in a database migration?

Reply:
I thought about it, but you need to have admin permissions
```
[info] SQL State  : 42501
[info] Error Code : 0
[info] Message    : ERROR: permission denied to create extension "uuid-ossp"
```